posts/entando-personas-make-modular-applications-more-manageable.md

```diff
@@ -0,0 +1,59 @@
+
+Curators also ensure that components are ready for the Composers on the team to use. They test for bugs and validate any business requirements. 
+
+Good communication skills are essential for Curators because they bridge the gap between Creators and Composers. They moderate content but should avoid being a bottleneck. Taking an attitude of, "let's figure out what is wrong" will likely lead to better results than rejecting components and leaving it up to the other team to fix the issues and bring the piece back for re-review.
+
+## Composers
+Composers work in the low-code environment of the Entando App Builder to combine the different business capabilities provided by the Creators and made available by the Curators to create an application that fits the business' needs. 
+
+A benefit of building your application in Entando is the Entando Hub, which contains numerous existing components that may fit your business case with little to no configuration required. The best code is often code your team does not need to write or maintain.
+
+Entando uses Role-Based Access Controls (RBAC), which restrict individual pages or directories  to people with the knowledge and business need to access them. Favoring components with this built-in functionality, or building components around RBAC from the beginning, will allow you to make sure that all users have the right level of access to all parts of your application without much extra work.
+
+## Consumers
+Consumers are the final step in the Application Composition process. They add and update web content within the page templates provided by the Composers, then set up analytics to track how that content performs. 
+
+Consumers are not necessarily restricted to managing content, however. They can create and manage users and roles, and edit rules and processes that run on the application. 
+Finally, Consumers can analyze application traffic and tune settings in the Entando Operator configuration to maximize performance.
+
+## In Conclusion
+With Personas, Entando provides a useful framework for every member of a team, depending on their technical specialties and business knowledge, to quickly find where they would be most comfortable and able to contribute. Creators build components, Curators validate and organize them, Composers combine them, and Consumers fill in the details, keep content up to date, and respond to feedback. 
+
+You can learn more about Entando by clicking [here](https://entando.com/), or more about JHipster [here.](https://www.jhipster.tech/) If you have any other questions, we would love to hear from you at [contact@ippon.tech](mailto:contact@ippon.tech). 
+
```

Comment:
This is really great @fuhsaz! I loved learning more about Entando's Personas over the course of this blog. Are there any diagrams that you can include to show the relationship between the 4 personas/ flow of development?